messages/en.json

```diff
@@ -390,8 +390,8 @@
 	"users": "Users",
 	"userviews": "Userviews",
 	"userviews-description": "Pageviews of all pages created by a user",
-	"userviews-edit-count-warning": "$1 has made $2 edits! Fetching pages created could take a while. Please be patient.",
-	"userviews-oversized-set": "$1 has created over $2 pages. For performance reasons, only the first $3 pages will be processed.",
+	"userviews-edit-count-warning": "{{GENDER:$1|$1}} has made $2 {{PLURAL:$2|edit|edits}}! Fetching pages created could take a while. Please be patient.",
```

Comment:
For this line and the next, I'm not quite sure if GENDER will work. Partly because I'm not sure if `$1` is a link to a userpage (in which case you would need something like `{{GENDER:$3|$1}}`) or just the username, but also because I don't know how jquery.i18n works with usernames on external tools like this.

Reply:
With that said, it won't hurt to have it in there, I'm just not sure it will work as intended.